test/searcher.cpp

```diff
@@ -6,6 +6,16 @@ namespace kiwix
 {
 
 TEST(Searcher, search) {
+	// test usage of zim::Searcher::getResult()
+	zim::Archive archive("./test/zimfile.zim");
+	zim::Searcher ftsearcher(archive);
+	zim::Query query;
+	query.setQuery("ray", false);
+	auto search = ftsearcher.search(query);
+
+	auto srs = search.getResults(5, 10);
+	ASSERT_EQ(srs.size(), 5);	// Fails! usage of getResults is wrong.
```

Comment:
It is better to have the test made with the expected result (and have the test failing) and explain in the commit that this is this specific test that will be fixed in next commit.

And please put this test in a new test, this is specific enough to have a test to check incremental searching.

Reply:
done!